### DISCUSSION TITLE
Week 3 Of Coding

### DISCUSSION THREAD

**Discussion by FlorianHeck2** [FlorianHeck2/codespaces-jupyter#3] [Announcements]
Hey Guys,
This week, i started coding on my own little game, a maze. Mainly, i tried to get the movement and Hitboxes working. Here a little Screenshot from the actual game state.
If you want to have a look at the code you can find it here: https://github.com/FlorianHeck2/codespaces-jupyter/tree/main/project
p.s.: the player is designed by myself :D 
<img width="960" alt="image" src="https://github.com/FlorianHeck2/codespaces-jupyter/assets/152873399/475ec184-930f-4860-bcb4-1d2df3e7d81a">
I know, that it isn't mouch you can see jet, but i'll keep you updated here so feel free to visit my blog often.

**Comment by Y4nnik** [comment]
Wooow, this looks so nice, can't wait till this game is on Steam so I can buy it.

**Reply by FlorianHeck2** [reply]
Can't wait to publish it.

**Comment by Justus2004** [comment]
Coole Idee als Projekt bin gespannt auf die nächsten Ergebnisse. Der Code dazu sieht gut aus und so als hättest du schon sehr viel Erfahrung in Python (also zumindest mehr als ich).

**Reply by FlorianHeck2** [reply]
Danke; Ich hab aber auch erst vor ca ner Woche in Python angefangen, wenn man ein Projekt findet, an dem man Spaß hat, dann geht das mit dem lernen echt schnell!

**Comment by laraanastasia** [comment]
This looks very cool! super excited to see what you will do! I love the player design
